docs/source/modules/modules.rst

```diff
@@ -6,50 +6,50 @@ OpenKAT consists of individual modules, each of which performs a subtask of the
 
 The source code and technical documentation is included for each module in its own github repository. This document refers to these repos and their documentation. If you have any questions, please contact the team, see 'Contact' in the readme of the NL-KAT-Coordination repo.
 
-Rocky
-=====
+Rocky - frontend
+================
 
 Rocky is the front end for OpenKAT and based on a Django framework. Through Rocky, OpenKAT can be accessed. It provides a user interface for all information present in the system. It is also possible to access OpenKAT only through the APIs.
 
-`NL-KAT-rocky <https://github.com/minvws/nl-kat-rocky>`_ is the repository containing Rocky's source code and documentation.
+`Rocky <https://github.com/minvws/nl-kat-coordination/tree/main/rocky>`_ is the folder containing Rocky's source code and documentation.
 
-Mula
-====
+Mula - scheduler
+================
 
 Mula is the scheduler, which controls OpenKAT. Mula has extensive technical documentation and design schematics.
 
-`NL-KAT-mula <https://github.com/minvws/nl-kat-mula>`_ is the repository containing Mula's source code and documentation.
+`Mula <https://github.com/minvws/nl-kat-coordination/tree/main/mula>`_ is the folder containing Mula's source code and documentation.
 
-Octopoes
-========
+Octopoes - datamodel
+====================
 
-Octopoes contains the data model and the XTDB with all objects. The documentation accompanying Octopoes covers the technical side of the logic behind OpenKAT and includes extensive information for people who want to work with the data model themselves.
+Octopoes contains the datamodel with object-types and the XTDB with all objects. The documentation accompanying Octopoes covers the technical side of the logic behind OpenKAT and includes extensive information for people who want to work with the data model themselves.
```

Comment:
`Data model` is two words in English as far as I know? https://en.wikipedia.org/wiki/Data_model